test/support/taxonomy_helper.rb

```diff
@@ -94,7 +98,7 @@ def draft_taxon_1
     {
       "title" => "About your organisation",
       "base_path" => "/about-your-organisation",
-      "content_id" => "draft_taxon_1",
+      "content_id" => draft_taxon_1_content_id,
```

Comment:
This doesn't match up with the `draft_taxon_1_content_id` method in this module, which I find confusing?

Reply:
I'm not sure what you mean, this aims to follow what's being done for the other taxons and their content_ids, eg `child_taxon_content_id`

Reply:
The `draft_taxon_1_content_id` method gives `"draft_taxon_1"` as the content id [1], which doesn't match the value here. Does that help clarify my confusion?

1; https://github.com/alphagov/whitehall/pull/3564/files/b3a19a50f5fcb9145b0b452d9b875063f5b92782..6d626ae9e2eb598877567e563136fc046b07a9b6#diff-e2203318ea7fdfd069f24a1c05980ce0R19

Reply:
I think I've just been reading it wrong, nothing to see here, sorry for the confusion.